frontend/packages/otr/src/components/publicInterpreter/listing/row/Cells.tsx

```diff
@@ -12,61 +12,118 @@ const mapRegionsToTextElements = (regions: Array<string>) =>
     .sort()
     .map((translation, k) => <Text key={k}>{translation}</Text>);
 
+const ContactInformationFields = ({
+  interpreter,
+}: {
+  interpreter: PublicInterpreter;
+}) => {
+  const { t } = useAppTranslation({
+    keyPrefix: 'otr.component.publicInterpreterListing.row',
+  });
+  const { email, phoneNumber, otherContactInfo } = interpreter;
+
+  return (
+    <>
+      {email && (
+        <div className="columns">
+          <div title={t('emailAriaLabel')}>
+            <MailOutline
+              aria-label={t('emailAriaLabel')}
```

Comment:
Näitä ei kai tarvinne tälle ikonille. Tai ainakin näyttäis, että noille tulee aria-hidden, joilloin niihin ei kai edes pääse näppiksellä. Tai vois tän vielä varmistaa Emmalta.

Reply:
Muokkasin tätä ratkaisua hieman